docs/proposals/config_data.md

```diff
@@ -173,7 +173,36 @@ type ConfigDataSelector struct {
 
 ### Volume Source
 
-The volume source will be addressed in a follow-up PR.
+A new `ConfigDataVolumeSource` type of volume source containing the `ConfigData` object will be
+added to the `VolumeSource` struct in the API:
+
+```go
+package api
+
+type VolumeSource struct {
+  // other fields omitted
+  ConfigData *ConfigDataVolumeSource `json:"configData,omitempty"`
+}
+
+// ConfigDataVolumeSource represents a volume that holds configuration data
+type ConfigDataVolumeSource struct {
+  // A list of config data keys to project into the volume in files
+  Files []ConfigDataVolumeFile `json:"files"`
```

Comment:
oh hell, now I don't know.  A sensible name now but inconsistent vs consistent but to be changed later.  I can go either way.